chef_master/source/about_chefdk.rst

```diff
@@ -26,10 +28,10 @@ Chef is a systems and cloud infrastructure automation framework that makes it ea
 .. end_tag
 
 * `An Overview of Chef </chef_overview.html>`_
-* `Install the Chef DK </install_dk.html>`_
+* `Install the ChefDK </install_dk.html>`_
 * `Ruby Guide </ruby.html>`_
 
-.. note:: See this blog post by Irving Popovetsky about running the Chef DK on Windows: https://www.chef.io/blog/2014/11/04/the-chefdk-on-windows-survival-guide/.
+.. note:: See this blog post by Irving Popovetsky about running the ChefDK on Windows: https://www.chef.io/blog/2014/11/04/the-chefdk-on-windows-survival-guide/.
```

Comment:
could we have the link text for that link be `blog post about running ChefDK on Windows` or similar instead of using the url as the link text?

Reply:
Sure! Anything is possible! 😄